flight/Modules/Battery/battery.c

```diff
@@ -133,6 +140,7 @@ static void batteryTask(void * parameters)
 		if (voltageADCPin >= 0) {
 			float adc_voltage = (float)PIOS_ADC_GetChannelVolt(voltageADCPin);
 			float scaled_voltage = 0.0f;
+			unsigned cells = batterySettings.NbCells;
```

Comment:
@tracernz yah so removing the static here broke it. Was there a reason for this change? No statics allowed?